### ISSUE TITLE
crmsh OBS repository for CentOS 7 missing python-parallax that is required by crmsh RPM

### ISSUE THREAD

**Issue by rlaurika** [ClusterLabs/crmsh#172]
When I try to install crmsh using the OBS repo I get this error:

```
Resolving Dependencies
--> Running transaction check
---> Package crmsh.noarch 0:2.3.2-1.1 will be installed
--> Processing Dependency: crmsh-scripts >= 2.3.2-1.1 for package: crmsh-2.3.2-1.1.noarch
--> Processing Dependency: python-parallax for package: crmsh-2.3.2-1.1.noarch
--> Running transaction check
---> Package crmsh.noarch 0:2.3.2-1.1 will be installed
--> Processing Dependency: python-parallax for package: crmsh-2.3.2-1.1.noarch
---> Package crmsh-scripts.noarch 0:2.3.2-1.1 will be installed
--> Finished Dependency Resolution
Error: Package: crmsh-2.3.2-1.1.noarch (opensuse-ha-clustering)
           Requires: python-parallax
 You could try using --skip-broken to work around the problem
 You could try running: rpm -Va --nofiles --nodigest
```

This is the yum repository I am using:
http://download.opensuse.org/repositories/network:/ha-clustering:/Stable/CentOS_CentOS-7/

Based on a previously installed system the python-parallax package used to be installed from the same repository as crmsh:

```
Installed Packages
Name        : python-parallax
Arch        : noarch
Version     : 1.0.1
Release     : 14.1
Size        : 103 k
Repo        : installed
From repo   : opensuse-ha-clustering
Summary     : Execute commands and copy files over SSH to multiple machines at
            : once
URL         : https://github.com/krig/parallax/
Licence     : BSD-3-Clause
Description : Parallax SSH provides an interface to executing commands on
            : multiple nodes at once using SSH. It also provides commands for
            : sending and receiving files to multiple nodes using SCP.
```

**Comment by krig** [comment]
Seems the package build was broken for some reason, should be fixed now.

**Comment by rlaurika** [comment]
@krig Thanks for the quick response. It looks like there is still a problem with the signature of the python-parallax package. This is on a freshly installed CentOS 7 virtual machine:

```
Error: Execution of '/usr/bin/yum -d 0 -e 0 -y install crmsh' returned 1: warning: /var/cache/yum/x86_64/7/opensuse-ha-clustering/packages/python-parallax-1.0.1-22.1.noarch.rpm: Header V3 RSA/SHA256 Signature, key ID 17280ddf: NOKEY
Public key for python-parallax-1.0.1-22.1.noarch.rpm is not installed


Error downloading packages:
  crmsh-scripts-2.3.2-1.1.noarch: [Errno 256] No more mirrors to try.
  crmsh-2.3.2-1.1.noarch: [Errno 256] No more mirrors to try.
```

**Comment by krig** [comment]
Yeah, there is a problem with the OBS, mirrors are publishing bad packages: I have emailed them about this.

**Comment by rlaurika** [comment]
Seems to be working now. Thanks!

**Comment by krig** [comment]
Great!

**Comment by nwarriorch** [comment]
Is this issue closed?. Python-Parallax is still not present in centos 6 repos. 
http://download.opensuse.org/repositories/network:/ha-clustering:/Stable/CentOS_CentOS-6/

**Comment by krig** [comment]
Apologies, I didn't have time to figure out why it was still failing. I've
fixed it now.

**Comment by bennetts2** [comment]
Sorry to reopen a year-old ticket, but did this ever get fixed properly?
I'm trying this on a Fedora 26 box, and I get:
```
Error: 
 Problem: conflicting requests
  - nothing provides python-parallax needed by crmsh-3.0.0-6.1.noarch
```

**Comment by krig** [comment]
Should have been fixed now, and there's a python-parallax package in the repository. Might be a new problem.

**Comment by bennetts2** [comment]
Sorry, I'm looking in the Fedora 26 repository and I don't see it there. Maybe I should have opened a separate ticket, but it sounded like the same thing. I'll try and use the CentOS RPMs for now.

**Comment by krig** [comment]
Ah, the Fedora 26 repository is probably not being rebuilt anymore for some reason. I am not great at keeping those up to date.. I would recommend trying the CentOS RPMs, they ought to work for Fedora too.

**Comment by bennetts2** [comment]
The CentOS RPMs don't seem to want to install on Fedora 26, DNF claims that "nothing provides /bin/python". The SRPMs seem to build perfectly though, so I've done a local build of the RPMs and installed those. I'll just have to pay attention to updates now.
